python/ray/serve/api.py

```diff
@@ -1172,65 +1196,113 @@ def get_handle(self, sync: Optional[bool] = True
                        ) -> Union[RayServeHandle, RayServeSyncHandle]:
             """Get a ServeHandle to this deployment."""
             return _get_global_client().get_handle(
-                Deployment._name, missing_ok=False, sync=sync, _internal=True)
+                Deployment._name, missing_ok=True, sync=sync, _internal=True)
 
         @classmethod
-        def options(self,
-                    backend_def: Optional[Callable] = None,
-                    name: Optional[str] = None,
-                    version: Optional[str] = None,
-                    ray_actor_options: Optional[Dict] = None,
-                    config: Optional[BackendConfig] = None) -> "Deployment":
+        def options(
+                self,
```

Comment:
Hadn't seen `@classmethod` before so I looked it up--it looks like it's idiomatic to for the first argument to be `cls` instead of `self` and to use `cls.<...>` instead of `Deployment.<...>`.  I'm not sure if there's any concrete benefit to doing so though, unless we plan to subclass Deployment or change its name in the future.

Reply:
Ah yeah really good point, that would be much cleaner IMO (it's weird to reference the class from within its own definition...)

Reply:
updated